### PR TITLE
Fix cache disk index error

### PR DESCRIPTION
**Pull request by xige-16**
issue: #20359 
/kind bug
Signed-off-by: xige-16 <xi.ge@zilliz.com>

### CODE REVIEW

**Comment by xige-16** [comment]
rerun ut

File: internal/core/src/common/Consts.h [hunk 1]
```diff
@@ -37,7 +37,7 @@ const char INDEX_BUILD_ID_KEY[] = "indexBuildID";
 const char INDEX_ROOT_PATH[] = "index_files";
 const char RAWDATA_ROOT_PATH[] = "raw_datas";
 
-const int DEFAULT_DISK_INDEX_MAX_MEMORY_LIMIT = 2;  // gigabytes
+const int64_t DEFAULT_DISK_INDEX_MAX_MEMORY_LIMIT = 2147483648;  // gigabytes
```
Comment (longjiquan) [inline internal/core/src/common/Consts.h:40]: rename `gigabytes` to `bytes`.

**Pull request merged**